### PR TITLE
Update profile page

### PR DESCRIPTION
**Pull request by guilgaly**

### CODE REVIEW

File: content/reference/user/profile/index.md [hunk 1]
```diff
@@ -9,17 +9,18 @@ weight: 10030
 
 ## Accessing your profile
 
-You can view your profile at any time by clicking on the **My Profile** button in the navigation bar.
+{{< img src="profile_menu.png" alt="My profile button" >}}
 
-{{< img src="profile.png" alt="Profile" >}}
+You can view your profile at any time by clicking on the **My profile** button in the navigation bar.
```
Comment (guilgaly) [inline content/reference/user/profile/index.md:14]: There may be a better way to describe where this button is... Do we have a name for the drop-down menu on the organization avatar?

Comment (ccedric) [reply content/reference/user/profile/index.md:14]: on the organization button ar the top-right of the screen ?

File: content/reference/user/profile/index.md [hunk 2]
```diff
@@ -9,17 +9,18 @@ weight: 10030
...
 - First and Last name
-- Mail address
-- Different roles
+- Email address
+- Your roles (for all teams and for specific teams)
 
 To update your profile, modify the fields you want to change, then click on the button **Update your information**.
 
-You can't update your username here. You need to ask a System Admin to change your roles on the **Users Admin** page.
+You cannot change your GitHub username or your roles here. You need to ask a System Admin to change them in the **Users Admin** page.
```
Comment (ccedric) [inline content/reference/user/profile/index.md:26]: You could add something like this: To know who are your system admins, please check the organization page.

File: content/reference/user/profile/index.md [hunk 3]
```diff
@@ -9,17 +9,20 @@ weight: 10030
 
 ## Accessing your profile
 
-You can view your profile at any time by clicking on the **My Profile** button in the navigation bar.
+{{< img src="profile_menu.png" alt="My profile button" >}}
 
-{{< img src="profile.png" alt="Profile" >}}
+You can view your profile at any time by clicking on the **My profile** button, from the drop-down menu available by clicking
```
Comment (Isammoc) [inline content/reference/user/profile/index.md:14]: My profile is not on top right dropdown (organization now) but in the left bar

File: content/reference/user/profile/index.md [hunk 4]
```diff
@@ -9,17 +9,20 @@ weight: 10030
...
 - First and Last name
-- Mail address
-- Different roles
+- Email address
+- Your roles (for all teams and for specific teams)
 
 To update your profile, modify the fields you want to change, then click on the button **Update your information**.
 
-You can't update your username here. You need to ask a System Admin to change your roles on the **Users Admin** page.
+You cannot change your GitHub username or your roles here. You need to ask a System Admin to change them in the **Users Admin** page.
```
Comment (Isammoc) [inline content/reference/user/profile/index.md:27]: Username cannot should not be changed by anyone for now

File: content/reference/user/profile/index.md [hunk 5]
```diff
@@ -9,17 +9,20 @@ weight: 10030
...
+You can see the list of you System Admins on the **Organizations** page.
```
Comment (Isammoc) [inline content/reference/user/profile/index.md:28]: Your

Comment (Isammoc) [reply content/reference/user/profile/index.md:28]: (drop down top right)

File: content/reference/user/profile/index.md (hunk 3 above)
Comment (guilgaly) [reply content/reference/user/profile/index.md:14]: It is in the top right menu. I even added a screenshot to show where it is.

Comment (Isammoc) [reply content/reference/user/profile/index.md:14]: My bad (and I was on phone)
I was mislead by the Users icon selected in the left bar

File: content/reference/user/profile/index.md [hunk 6]
```diff
@@ -9,17 +9,20 @@ weight: 10030
```
Comment (Isammoc) [inline content/reference/user/profile/index.md:9]: profile_menu.png => My Org
(Big view)

File: content/reference/user/profile/index.md [hunk 7]
```diff
@@ -9,17 +9,20 @@ weight: 10030
...
 - First and Last name
-- Mail address
-- Different roles
+- Email address
+- Your roles (for all teams and for specific teams)
 
 To update your profile, modify the fields you want to change, then click on the button **Update your information**.
 
-You can't update your username here. You need to ask a System Admin to change your roles on the **Users Admin** page.
+You cannot change your your roles here. You need to ask a System Admin to change them in the **Users Admin** page. You can see
```
Comment (Isammoc) [inline content/reference/user/profile/index.md:27]:
```suggestion
You cannot change your roles here. You need to ask a System Admin to change them in the **Users Admin** page. You can see
```

Perhaps a link to the page to manage Users?

File: content/reference/user/profile/index.md [hunk 8]
```diff
@@ -9,17 +9,20 @@ weight: 10030
...
+the list of your System Admins on the **Organization** page, accessible from the drop-down menu at the top-right of the screen.
```
Comment (franciscotco) [inline content/reference/user/profile/index.md:28]: Perhaps a link to the organization page from the doc?

**Review by Isammoc — APPROVED** [review]

**Pull request merged**